crypto/objects/obj_dat.h

```diff
@@ -10,7 +10,7 @@
  */
 
 /* Serialized OID's */
-static const unsigned char so[8487] = {
+static const unsigned char so[8504] = {
```

Comment:
Hmm, when declared as `so[]` `sizeof(so)` gives `8503` - so we have been off-by-one for a while here. Is the extra byte needed as a null terminator? Or would it be best just to use `... so[] = {`?

Reply:
Or is there a reason to size this at one more than the number of bytes?

Reply:
(also, `so[]` and `nid_objs[]` are screaming out to be generated by a script, but that's for another time)